specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -3956,6 +4152,329 @@
       },
       "description": "Response from a List Indexes request. If successful, it includes the full definitions of all indexes."
     },
+	"Skillset": {
+		"properties": {
+			"name": {
+				"externalDocs": {
+					"url": "https://docs.microsoft.com/en-us/azure/search/cognitive-search-tutorial-blob"
+				},
+				"type": "string",
+				"description": "The name of the skillset."
+			},
+			"description": {
+				"type": "string",
+				"description": "The description of the skillset."
+			},
+			"skills": {
+				"type": "array",
+				"items": {
+					"$ref": "#/definitions/Skill"
+				},
+				"description": "A list of skills in the skillset."
+			},
+			"@odata.etag": {
+				"x-ms-client-name": "ETag",
+				"type": "string",
+				"description": "The ETag of the skillset."
+			}
+		},
+		"required": ["name",
+		"description",
+		"skills"],
+		"description": "A list of Cognitive Search skills."
+	},
+	"Skill": {
+		"discriminator": "@odata.type",
+		"properties": {
+			"@odata.type": {
+				"type": "string"
+			},
+			"description": {
+				"type": "string",
+				"description": "The description of the skill."
+			},
+			"context": {
+				"type": "string",
+				"description": "The context of the skill."
+			},
+			"inputs": {
+				"type": "array",
+				"items": {
+					"$ref": "#/definitions/InputFieldMappingEntry"
+				},
+				"description": "The inputs of the skill."
+			},
+			"outputs": {
+				"type": "array",
+				"items": {
+					"$ref": "#/definitions/OutputFieldMappingEntry"
+				},
+				"description": "The outputs of the skill."
+			}
+		},
+		"required": ["@odata.type",
+		"description",
+		"context",
+		"inputs",
+		"outputs"],
+		"description": "Abstract base class for skills."
+	},
+	"InputFieldMappingEntry": {
+		"properties": {
+			"name": {
+				"type": "string",
+				"description": "The name of the input."
+			},
+			"source": {
+				"type": "string",
+				"description": "The source of the input."
+			}
+		},
+		"required": ["name",
+		"source"],
+		"description": "Input field mapping for a skill."
+	},
+	"OutputFieldMappingEntry": {
+		"properties": {
+			"name": {
+				"type": "string",
+				"description": "The name of the output."
+			},
+			"targetName": {
+				"type": "string",
+				"description": "The target name of the output."
+			}
+		},
+		"required": ["name",
+		"targetName"],
+		"description": "output field mapping for a skill."
+	},
+	"KeyPhraseExtractionSkill": {
+		"x-ms-discriminator-value": "#Microsoft.Skills.Text.KeyPhraseExtractionSkill",
+		"allOf": [{
+			"$ref": "#/definitions/Skill"
+		}],
+		"properties": {
+			"defaultLanguageCode": {
+				"type": "string",
+				"default": "en",
+				"description": "A value indicating which language code. Default is en."
```

Comment:
Is there an enum that lists all the possible values for this property?

Reply:
Synced up with Bruce offline and use the extensible enums for the supported language codes for each skill.